### DISCUSSION TITLE
迅雷云盘的链接出了问题

### DISCUSSION THREAD

**Discussion by SeanAshe** [AlistGo/alist#4634] [Q&A]
2023/06/22 01:31:31.077031 ERROR RESTY Get "https://api-pan.xunlei.com/drive/v1/files?__sync=true&__type=drive&filters=%7B%22phase%22%3A%7B%22eq%22%3A%22PHASE_TYPE_COMPLETE%22%7D%2C%22trashed%22%3A%7B%22eq%22%3Afalse%7D%7D&limit=100&page_token=&parent_id=VMUfpkyNepj25NtsaLSRMFS4A1&refresh=true&space=&with_audit=true": context deadline exceeded (Client.Timeout exceeded while awaiting headers), Attempt 1
2023/06/22 01:33:58.172094 ERROR RESTY Post "https://xluser-ssl.xunlei.com/v1/shield/captcha/init": net/http: TLS handshake timeout, Attempt 1
2023/06/22 01:40:22.331208 ERROR RESTY Get "https://api-pan.xunlei.com/drive/v1/files/VNTdM9nYLuKmgv7UvJLXZOtIA1": net/http: TLS handshake timeout, Attempt 1
2023/06/22 01:52:18.399236 ERROR RESTY Get "https://api-pan.xunlei.com/drive/v1/files/VNYGksGk6ZYRbYP_pGVKJk0BA1": net/http: TLS handshake timeout, Attempt 1
2023/06/22 01:52:28.521263 ERROR RESTY Get "https://api-pan.xunlei.com/drive/v1/files/VNYGksGk6ZYRbYP_pGVKJk0BA1": net/http: TLS handshake timeout, Attempt 2
2023/06/22 01:54:42.232207 ERROR RESTY Get "https://api-pan.xunlei.com/drive/v1/files/VNTdM9nYLuKmgv7UvJLXZOtIA1": net/http: TLS handshake timeout, Attempt 1
2023/06/22 01:54:44.283282 ERROR RESTY Get "https://api-pan.xunlei.com/drive/v1/files/VNNqb9NEv_gDeb1GulsBaSmdA1": net/http: TLS handshake timeout, Attempt 1
2023/06/22 01:54:52.355333 ERROR RESTY Get "https://api-pan.xunlei.com/drive/v1/files/VNTdM9nYLuKmgv7UvJLXZOtIA1": net/http: TLS handshake timeout, Attempt 2
2023/06/22 01:59:04.706468 ERROR RESTY Post "https://xluser-ssl.xunlei.com/v1/shield/captcha/init": net/http: TLS handshake timeout, Attempt 1
2023/06/22 01:59:06.706468 ERROR RESTY Post "https://xluser-ssl.xunlei.com/v1/shield/captcha/init": net/http: TLS handshake timeout, Attempt 1
2023/06/22 02:06:22.695242 ERROR RESTY Get "https://api-pan.xunlei.com/drive/v1/files/VNYGksGk6ZYRbYP_pGVKJk0BA1": net/http: TLS handshake timeout, Attempt 1
2023/06/22 02:06:32.819370 ERROR RESTY Get "https://api-pan.xunlei.com/drive/v1/files/VNYGksGk6ZYRbYP_pGVKJk0BA1": net/http: TLS handshake timeout, Attempt 2
2023/06/22 02:06:42.965921 ERROR RESTY Get "https://api-pan.xunlei.com/drive/v1/files/VNYGksGk6ZYRbYP_pGVKJk0BA1": net/http: TLS handshake timeout, Attempt 3
2023/06/22 02:06:53.314644 ERROR RESTY Post "https://xluser-ssl.xunlei.com/v1/shield/captcha/init": net/http: TLS handshake timeout, Attempt 1
2023/06/22 02:07:03.431584 ERROR RESTY Post "https://xluser-ssl.xunlei.com/v1/shield/captcha/init": net/http: TLS handshake timeout, Attempt 2
2023/06/22 02:09:30.071380 ERROR RESTY Get "https://api-pan.xunlei.com/drive/v1/files/VNYGksGk6ZYRbYP_pGVKJk0BA1": net/http: TLS handshake timeout, Attempt 1
2023/06/22 02:09:40.199451 ERROR RESTY Get "https://api-pan.xunlei.com/drive/v1/files/VNYGksGk6ZYRbYP_pGVKJk0BA1": net/http: TLS handshake timeout, Attempt 2
2023/06/22 02:12:07.000122 ERROR RESTY Get "https://api-pan.xunlei.com/drive/v1/files/VNYGksGk6ZYRbYP_pGVKJk0BA1": net/http: TLS handshake timeout, Attempt 1
2023/06/22 02:14:33.175337 ERROR RESTY Get "https://api-pan.xunlei.com/drive/v1/files/VNYGksGk6ZYRbYP_pGVKJk0BA1": net/http: TLS handshake timeout, Attempt 1
2023/06/22 02:19:21.399228 ERROR RESTY Post "https://xluser-ssl.xunlei.com/v1/shield/captcha/init": net/http: TLS handshake timeout, Attempt 1
2023/06/22 02:19:31.524634 ERROR RESTY Post "https://xluser-ssl.xunlei.com/v1/shield/captcha/init": net/http: TLS handshake timeout, Attempt 2
2023/06/22 02:19:41.727284 ERROR RESTY Post "https://xluser-ssl.xunlei.com/v1/shield/captcha/init": net/http: TLS handshake timeout, Attempt 3
2023/06/22 02:19:52.041019 ERROR RESTY Post "https://xluser-ssl.xunlei.com/v1/shield/captcha/init": net/http: TLS handshake timeout, Attempt 4
2023/06/22 02:20:02.092530 ERROR RESTY Post "https://xluser-ssl.xunlei.com/v1/shield/captcha/init": net/http: TLS handshake timeout, Attempt 1
2023/06/22 02:20:12.217040 ERROR RESTY Post "https://xluser-ssl.xunlei.com/v1/shield/captcha/init": net/http: TLS handshake timeout, Attempt 2
2023/06/22 02:20:22.403197 ERROR RESTY Post "https://xluser-ssl.xunlei.com/v1/shield/captcha/init": net/http: TLS handshake timeout, Attempt 3
2023/06/22 02:22:52.119214 ERROR RESTY Get "https://api-pan.xunlei.com/drive/v1/files/VNYGksGk6ZYRbYP_pGVKJk0BA1": net/http: TLS handshake timeout, Attempt 1

**Comment by youtehub** [comment]
使用迅雷专家版挂载吧，填入用户名，密码即可，自动获取验证刷新令牌;或者升级到最新版看看
![image](https://img1.imgtp.com/2023/06/24/Rjri6Dkr.png)
